### ISSUE TITLE
Database failing to start, ~/.msf4/db/log does not exist

### ISSUE THREAD

**Issue by 0sx** [rapid7/metasploit-framework#8080]
Following the instructions [here](https://github.com/rapid7/metasploit-framework/wiki/Nightly-Installers#os-x-manual-installation) I installed the latest nightly installer, which was version 4.14.1, on macOS 10.12.3 build 16D32.  I ran

`/opt/metasploit-framework/bin/msfconsole`

and the database was successfully created but failed to start with the error

`No such file or directory @ rb_sysopen - /Users/0sx/.msf4/db/log (Errno::ENOENT)`

I confirmed this file did not exist.  If I try to do

`/opt/metasploit-framework/bin/msfdb reinit`

it fails with the same error.  The same for doing `delete` and then `init`.  If I manually create `/Users/0sx/.msf4/db/log` starting the database fails but I'm then asked if the database is corrupt would I like to reinitialize it.  However this deletes everything in the `db` folder and I'm back to having no `log` file.

`production.log` was empty and the `sessions` directory didn't contain anything.  The `framework.log` lists the failed attempts to connect to the databse.

I've not run this as root and have ensured the user owns those folders.  As a precaution I have also done `chown -R 0sx ~/.msf4` before trying the above steps again but with no change.

**Comment by 0sx** [comment]
After much frustration and digging I discovered that the call to `/opt/metasploit-framework/embedded/bin/initdb` was failing because I had changed text encoding to Latin 1 and it required UTF-8 to test something else.  After changing it back to UTF-8 I manually called `initdb` with the arguments `msfdb` does in `create_db`.  The role msf did not exist so I ran `msfdb reinit` and `msfconsole` then started without any issues and `db_status` showed as connected.
